data_structures/arrays/permutations.py

```diff
@@ -0,0 +1,31 @@
+from itertools import permutations
```

Comment:
If this is not being used in the module, please remove it. I see it's just used in the `if __name__ == ...` block, add it there instead.